src/topology/perfect.lean

```diff
@@ -5,6 +5,7 @@ Authors: Felix Weilacher
 -/
 import topology.separation
 import topology.bases
+import topology.scheme
```

Comment:
This is a bit of an unfortunate situation but by making the import go this way you're promising to port `topology.scheme` and [all its unported dependencies](https://leanprover-community.github.io/mathlib-port-status/file/topology/metric_space/pi_nat) yourself (well, still using mathport) to mathlib4. Not saying you should change it, just letting you know.

Reply:
I'm not sure I understand. It seems like you linked the dependencies for `metric_space.pi_nat`. After this pr, the only dependency of `topology.scheme` (or whatever we change it to) should be `topology.perfect`.

Reply:
> After this pr, the only dependency of `topology.scheme` (or whatever we change it to) should be `topology.perfect`. 

Did you mean it the other way around? Currently this PR makes `topology.perfect` import `topology.scheme`?

Reply:
Yes I guess I meant dependent? I'm not used to the lingo haha.

Reply:
I see, so you're saying that because `topology.scheme` imports `pi_nat`, I'd have to port `pi_nat` and everything it imports, etc? I am not necessarily objecting, but why would I have to do it instead of any of the other people importing `pi_nat` or its dependencies?

Reply:
It's because `topology.perfect` is already ported.

Reply:
Oh I had no idea. How difficult is porting?

Reply:
Not very hard. You'll have to follow the instructions over the link I sent you.

Reply:
I can give it a go.

Reply:
To port this new file itself, you will have to wait for the day after this PR has been merged. The rest should already be portable now.

Reply:
`pi_nat` and all its dependencies have now been ported (not by me, lol). 

So what is the process now? Do I need to port the contents of this PR to mathlib4 before it can be merged in mathlib3?

Reply:
No, you get this PR merged, then you head over to the [port dashboard page of your new file](https://leanprover-community.github.io/mathlib-port-status/file/topology/metric_space/scheme) (will be up once this PR has been merged) and follow the instructions.